examples/aurelia/.gitignore

```diff
@@ -0,0 +1,11 @@
+node_modules/*
```

Comment:
All TodoMVC apps should run without the user having to install npm packages or building anything so please remove node_modules from your .gitignore

Reply:
The exampled should be committed with all the files it needs to run

Reply:
@FadySamirSadek The following lines of that gitignore file should ensure user have all the necessary files to run the app, unless I misunderstood the git ignore syntax

Reply:
You're right, my bad

Reply:
LGTM